graphql-jpa-query-schema/src/test/java/com/introproventures/graphql/jpa/query/converter/GraphQLJpaConverterTests.java

```diff
@@ -828,7 +828,985 @@ public void queryTasksVariablesWhereWithEXISTSByNameAndValueCriteria() {
 
         // then
         assertThat(result.toString()).isEqualTo(expected);
-    }       
-    
-         
+    }
+
+    @Test
```

Comment:
@SinaiNIN Can you extract your tests into a separate a package with its own test class, data.sql and test model?

Reply:
Yes, I can. It would be better.